helm/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for ack-sagemaker-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: mbaijal/ack
```

Comment:
I'm not sure if this is a good idea that we should push it into the official Github repository. One alternative is that you provide a values override file. As part of canaries or tests you can download and access these helm charts and then submit, when applying the chart to your cluster, with another values file overriding this value.
See https://stackoverflow.com/questions/51097553/can-i-have-multiple-values-yaml-files-for-helm/51099510 for information about applying an alternate values file when installing a chart.